packages/fluentui/react-northstar/src/themes/teams/types.ts

```diff
@@ -149,7 +149,7 @@ export type TeamsThemeStylesProps = {
   ToolbarMenuRadioGroup: ToolbarMenuRadioGroupStylesProps;
   TooltipContent: TooltipContentStylesProps;
   Text: TextStylesProps;
-  TextAre: TextAreStylesProps;
+  TextArea: TextAreaStylesProps;
```

Comment:
Shouldn't we mention that as breaking change as well, since we are changing the exported name?

Reply:
Not sure, we have been doing this in all conversions and I think we haven't mentioned? If we should I can just add it